runtime/pom.xml

```diff
@@ -44,6 +44,17 @@
             <scope>provided</scope>
             <optional>true</optional>
         </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-hibernate-orm-panache</artifactId>
+            <scope>provided</scope>
+            <optional>true</optional>
+        </dependency>
```

Comment:
Why is this in runtime and not in the tests?

Reply:
Otherwise we require another maven module to test it ?

Reply:
Ah wait, now I got it. It is marked as provided and optional, so it does not end in the jar.
But would marking it as test dependency solve it too?